packages/zosfiles/src/cli/ZosFilesBase.handler.ts

```diff
@@ -37,6 +37,7 @@ export abstract class ZosFilesBaseHandler implements ICommandHandler {
 
         const response = await this.processWithSession(commandParameters, session, profile);
 
+        commandParameters.response.progress.endBar(); // end any progress bars
```

Comment:
Does there need to be a similar statement in ZosmfBaseHandler? Also, do we terminate any outstanding progress bars in the event of an error in the command processor?

Reply:
If it's being done under error conditions (and success) in Imperative, I think we're covered. Thanks.